app/helpers/application_helper.rb

```diff
@@ -1042,6 +1042,25 @@ def perfmenu_click?
   end
 
   def javascript_redirect(args)
+    if args.class == Hash && args[:action] == 'show' && restful_routed?(self.class.model)
+      args.delete(:action)
+      polymorphic_path_redirect(model, args)
+    else
+      non_polymorphic_path_redirect(args)
+    end
+  end
+
+  def polymorphic_path_redirect(model, args)
+    args[:record] ? record = args[:record] : record = model.find(args[:id] || params[:id])
+    args.delete(:record)
+    args.delete(:id)
+    render :update do |page|
```

Comment:
@martinpovolny Currently we have close to 285 `render :update do |page|` references.
```
$ git grep "render :update do |page|" |wc -l
     285

```
Is there a PR currently that would address these or are you referring to a future PR?